lib/widgets/genre_list.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+
+class GenreList extends StatelessWidget {
+  const GenreList({
+    super.key,
+    required this.genres,
+  });
+
+  static const double height = 70;
+  final List<String> genres;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: height,
+      child: ListView(
+        scrollDirection: Axis.horizontal,
+        physics: const ScrollPhysics(),
+        children: [
+          for (String item in genres) GenreListItem(item: item),
+        ],
+      ),
+    );
+  }
+}
+
+class GenreListItem extends StatelessWidget {
+  static const Color cardColor = Colors.black45;
+  static const double cardBorderRadius = 3;
+  static const Color cardBorderColor = Colors.black;
+  static const double cardBorderWidth = 2;
+  static const double listPadding = 4;
+  static const Color cardTextColor = Colors.white70;
+
+  const GenreListItem({
+    super.key,
+    required this.item,
+  });
+
+  final String item;
+
+  @override
+  Widget build(BuildContext context) {
+    return FittedBox(
+      child: Card(
+        color: cardColor,
+        child: Container(
+          decoration: BoxDecoration(
+            borderRadius:
+                const BorderRadius.all(Radius.circular(cardBorderRadius)),
+            border: Border.all(
+              color: cardBorderColor,
+              width: cardBorderWidth,
+            ),
+          ),
+          child: Padding(
+            padding: const EdgeInsets.all(listPadding),
+            child: Text(
+              item,
+              style: const TextStyle(
+                color: cardTextColor,
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
```

Comment:
Here you don't need to separate in two different StatelessWidget

Reply:
I understand it's not necessary since the functionality is coupled anyways, however I do think it makes the code clearer. I got the idea to separate them from [this previous PR comment](https://github.com/jere53/jeremias.salinas/pull/1#discussion_r1291556215), and I tend to agree. Is this a matter where I can choose whichever style suits me? Or is it straight up better to keep all the functionality in the same widget?